app/assets/javascripts/covid19/src/webapp/components/SVGProteoma.tsx

```diff
@@ -0,0 +1,498 @@
+import React from "react";
+import styled from "styled-components";
+import _ from "lodash";
+import i18n from "../../utils/i18n";
+import { sendAnalytics } from "../../utils/analytics";
+
+interface SVGProteomaProps {
+    search: string;
+    setSearch: (value: string) => void;
+    visible: { orf1a?: boolean; orf1b?: boolean };
+    setVisible: (visible: { orf1a?: boolean; orf1b?: boolean }) => void;
```

Comment:
The type `{ orf1a?: boolean; orf1b?: boolean }` is repeated 3 times in the app, let's create a type alias.